microservices/cart/src/queries/services.py

```diff
@@ -26,6 +26,7 @@ class CartQueryService(QueryService):
     repository: CartQueryRepository = Provide["cart_repository"]
 
     @enroute.rest.query("/carts/{uuid}", "GET")
+    @enroute.broker.query("GetCartQRS")
```

Comment:
We need to discuss about the better strategy to difference the "snapshot" getters from the "query service" ones, but I think this is a good starting point. 😉